sw/airborne/firmwares/rotorcraft/main_fbw.c

```diff
@@ -175,9 +178,17 @@ STATIC_INLINE void main_periodic(void)
     }
   }
 
+
+  static uint16_t dv = 0;
+  // TODO make module out of led blink?
   /* set failsafe commands     */
   if (fbw_mode == FBW_MODE_FAILSAFE) {
     SetCommands(commands_failsafe);
+    if (!(dv++ % (PERIODIC_FREQUENCY / 20))) { LED_TOGGLE(3);}
```

Comment:
Just creating a makefile variable doesn't make a C define magically appear.
You need to add in the intermcu makefile something like for the RADIO_CONTROL_LED.

And it would probably be better to name this led something like FBW_MODE_LED